### PR TITLE
loss wrapper to batch dict

### PR DESCRIPTION
**Pull request by avihu111**
wraps a loss module to use the batch dict.

### CODE REVIEW

**Review by akoseki2 — APPROVED** [review]

**Comment by avihu111** [comment]
I added a usage example, hoping for faster code understanding.
@akoseki2 would you have another look please?

File: fuse/dl/losses/loss_wrap_to_dict.py [hunk 1]
```diff
@@ -0,0 +1,51 @@
+from typing import Dict
+import torch
+from torch import nn
+from fuse.utils.ndict import NDict
+
+
+class LossWrapToDict(torch.nn.Module):
+    """
+    Wraps a torch loss function to support a batch dict
```
Comment (mosheraboh) [inline fuse/dl/losses/loss_wrap_to_dict.py:9]: Lets mention the usage_example

**Review by mosheraboh — CHANGES_REQUESTED** [review]
Thanks @avihu111, looks good!
See few minor requests inline

**Review by akoseki2 — APPROVED** [review]

**Review by mosheraboh — APPROVED** [review]
